scripts/zones/Windurst_Waters/npcs/Honoi-Gomoi.lua

```diff
@@ -68,12 +79,17 @@ function onEventFinish(player, csid, option)
         player:setCharVar("CryingOverOnions", 1)
     elseif csid == 775 and npcUtil.giveItem(player, 13136) then
         player:confirmTrade()
-        player:setCharVar("CryingOverOnions", 2)
-    elseif csid == 776 and npcUtil.completeQuest(player, WINDURST, tpz.quest.id.windurst.CRYING_OVER_ONIONS, {fame=120, var="CryingOverOnions"}) then
+        player:setCharVar("CryingOverOnions", 3)
+    elseif csid == 776 and npcUtil.completeQuest(player, WINDURST, tpz.quest.id.windurst.CRYING_OVER_ONIONS, {
+        fame=120,
+        var="CryingOverOnions"
+    }) then
         player:needToZone(true)
     elseif csid == 780 then
         player:addQuest(WINDURST, tpz.quest.id.windurst.WILD_CARD)
-    elseif csid == 782 and npcUtil.completeQuest(player, WINDURST, tpz.quest.id.windurst.WILD_CARD, {title=tpz.title.DREAM_DWELLER, fame=135, var="WildCard"}) then
+    elseif csid == 782 and npcUtil.completeQuest(player, WINDURST, tpz.quest.id.windurst.WILD_CARD, {
```

Comment:
when we make a conditional span multi line, I think its easier to read when we put a newline at both ends of the condition and align the `then` to the start:

```lua
    elseif
        csid == 782 and 
        npcUtil.completeQuest(player, WINDURST, tpz.quest.id.windurst.WILD_CARD, {
            title=tpz.title.DREAM_DWELLER, fame=135, var="WildCard"
        })
    then
```

Reply:
Options of `npcUtil.completeQuest()` on one line or listed?

Reply:
Changed in accordance with **Inline tables** section from [Style Guide](https://github.com/project-topaz/topaz/blob/release/CONTRIBUTING.md).

Reply:
break at curly brace is what the existing examples and guide did/do